Dockerfiles/manifests/agent.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   name: datadog-agent
   namespace: default
 spec:
+  minReadySeconds: 0
+  updateStrategy:
+    rollingUpdate:
+      maxUnavailable: 25%
```

Comment:
I think 25% is a bit aggressive for a rolling update. what do you think about 10%